build/doc.go

```diff
@@ -22,7 +22,8 @@ a go development environment or using existing one for this project is not desir
 Available commands are:
 
 	install        -- install the dst-go software
-	lint           -- run meta-linter with configuration linterConfig.json
+	lint           -- run meta-linter with configuration as defined in the
+					  .golangci.yml file in project root.
```

Comment:
I'd prefer a short one-line message as in the original version, i.e. something like
```
lint           -- run meta-linter with configuration 'dst-go/.golangci.yml'
```

Reply:
Changes updated in this [commit](https://github.com/direct-state-transfer/dst-go/pull/14/commits/49c73432690b6042f1b8fd998b95db4f8a0d4a5e). I have also updated the message for runWalkthrough target to fit in one line.